__tests__/app.test.js

```diff
@@ -182,3 +181,51 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("should post a comment to an article by specific article_id", () => {
+    const commentData = {
+      username: "butter_bridge",
+      body: "I done 10 push-ups",
+    };
+    return request(app)
+      .post(`/api/articles/1/comments`)
+      .send(commentData)
+      .expect(201)
+      .then((response) => {
+        const commentArray = response.body;
+
+        expect(commentArray.comment).toMatchObject({
+          body: expect.any(String),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          article_id: expect.any(Number),
+          created_at: expect.any(String),
```

Comment:
We do know for a fact what some of these properties should be - e.g article_id, comment body, votes

It's good practice to only expect any type when you can't predict what the values will be exactly - when you can be specific, go for it!